modules/distribution/src/repository/resources/dbscripts/apimgt/oracle.sql

```diff
@@ -514,13 +516,13 @@ CREATE OR REPLACE TRIGGER IDP_ROLE_MAPPING_TRIG
                END;
 /
 CREATE TABLE IDP_CLAIM (
-            ID INTEGER,
-            IDP_ID INTEGER,
-            TENANT_ID INTEGER,
-            CLAIM VARCHAR(254),
-            PRIMARY KEY (ID),
-            UNIQUE (IDP_ID, CLAIM),
-            FOREIGN KEY (IDP_ID) REFERENCES IDP(ID) ON DELETE CASCADE)
+			ID INTEGER,
+			IDP_ID INTEGER,
+			TENANT_ID INTEGER,
+			CLAIM VARCHAR(254),
+			PRIMARY KEY (ID),
+			UNIQUE (IDP_ID, CLAIM),
+			FOREIGN KEY (IDP_ID) REFERENCES IDP(ID) ON DELETE CASCADE)
```

Comment:
Directly copied content from https://github.com/wso2/carbon-apimgt/tree/v5.0.4/features/apimgt/org.wso2.carbon.apimgt.core.feature/src/main/resources/sql as apim version upgraded to v5.04